### PR TITLE
Beepsky Non-Human Target Bugfix

### PR DESCRIPTION
**Pull request by SonixApache**
beepsky now properly targets non-humans and removes an old MLP reference that went very stale

## Changelog
 * bugfix: Officer Beepsky properly targets non-humans again

### CODE REVIEW

File: code/game/machinery/bots/secbot.dm [hunk 1]
```diff
@@ -381,7 +381,7 @@ Auto Patrol: []"},
 
 /obj/machinery/bot/secbot/proc/speak(var/message)
 	visible_message("<span class='game say'><span class='name'>[src]</span> beeps, \"[message]\"",\
-		drugged_message="<span class='game say'><span class='name'>[src]</span> beeps, \"[pick("Wait! Let's be friends!","Wait for me!","You're so cool!","Who's your favourite pony?","I-It's not like I like you or anything...","Wanna see a magic trick?","Let's go have fun, assistant-kun~")]\"")
+		drugged_message="<span class='game say'><span class='name'>[src]</span> beeps, \"[pick("Wait! Let's be friends!","Wait for me!","You're so cool!","I-It's not like I like you or anything...","Wanna see a magic trick?","Let's go have fun, assistant-kun~")]\"")
```
Comment (Eneocho) [inline code/game/machinery/bots/secbot.dm:384]: what did he mean by this

Comment (toomykins) [reply code/game/machinery/bots/secbot.dm:384]: my little pony was 13 years ago

Comment (Eneocho) [reply code/game/machinery/bots/secbot.dm:384]: this just gives a reason to add pony stuff to the game
like how people post barney and pony stuff to annoy that one barneydude

Comment (SECBATON-GRIFFON) [reply code/game/machinery/bots/secbot.dm:384]: missed cheese

**Comment by Eneocho** [comment]
👎🏻 because of the unatomic anti-scootaloo change, otherwise good change

**Review by west3436 — APPROVED** [review]

**Pull request merged**